src/dotnet-interactive/CommandLine/CommandLineParser.cs

```diff
@@ -504,12 +505,7 @@ private static CompositeKernel CreateKernel(string defaultKernelName, FrontendEn
         kernel.AddKernelConnector(new ConnectNamedPipeCommand());
         kernel.AddKernelConnector(new ConnectSignalRCommand());
         kernel.AddKernelConnector(new ConnectStdIoCommand(startupOptions.KernelHost));
-
```

Comment:
why is this gone?

Reply:
This wasn't really doing anything useful outside tests. I moved these older diagnostic methods over to the tests and want to make some more general and internally consistent diagnostic improvements.